tests/cases/types/brush.slint

```diff
@@ -58,12 +58,12 @@ assert_eq!(t.get_color_brush().color(), green);
 
 ```js
 var t = new slint.Test({});
-assert.equal(t.color_brush, "#0000ffff");
+assert.equal(t.color_brush.toString(), "#0000ffff");
```

Comment:
Is that needed or is the conversion not implicit?

Reply:
It does it implicit on `assert` but not with `assert.equal`.